perfkitbenchmarker/providers/aws/aws_disk.py

```diff
@@ -33,12 +33,49 @@
 VOLUME_EXISTS_STATUSES = frozenset(['creating', 'available', 'in-use', 'error'])
 VOLUME_DELETED_STATUSES = frozenset(['deleting', 'deleted'])
 VOLUME_KNOWN_STATUSES = VOLUME_EXISTS_STATUSES | VOLUME_DELETED_STATUSES
+
+STANDARD = 'standard'
+GP2 = 'gp2'
+IO1 = 'io1'
+
 DISK_TYPE = {
-    disk.STANDARD: 'standard',
-    disk.REMOTE_SSD: 'gp2',
-    disk.PIOPS: 'io1'
+    # no need to translate 'standard' because the old and new names
```

Comment:
I just talked about this with Eric. I misunderstood how defaults work. The *benchmark* is what determines whether or not we create a disk spec, not the flags. All the flags do is modify properties of the benchmark's config.

Right now, you can create disks without passing any arguments if the benchmark wants them. But there will be a warning, because the disk types are specified using the old names. I will update the file `default_config_constants.yaml` to fix this.